src/reachy2_sdk/parts/mobile_base.py

```diff
@@ -173,6 +173,23 @@ def set_speed(self, x_vel: float, y_vel: float, rot_vel: float) -> None:
         )
         self._mobility_stub.SendDirection(req)
 
+    def translate_by(self, x: float, y: float, timeout: Optional[float] = None) -> None:
+        """Send a target position relative to the current position of the mobile base.
+
+        (x, y) define the translation wanted in the mobile base in cartesian space."""
+        odometry = self.odometry
+        x = odometry["x"] + x
+        y = odometry["y"] + y
+        self.goto(x, y, 0, timeout=timeout)
+
+    def rotate_by(self, theta: float, timeout: Optional[float] = None) -> None:
+        """Send a target rotation relative to the current rotation of the mobile base.
+
+        theta defines the rotation wanted in the mobile base in cartesian space."""
+        odometry = self.odometry
+        theta = odometry["theta"] + theta
```

Comment:
We should add the position, or the robot will translate to get back to x=0, y = 0 if it already has been moved from its origin.
 <code> def rotate_by(self, theta: float, timeout: Optional[float] = None) -> None:
        odometry = self.odometry
        x = odometry["x"]
        y = odometry["y"]
        theta = odometry["theta"] + theta
        self.goto(x, y, theta, timeout=timeout) </code>